### ISSUE TITLE
UX/UI - V2 of Application questions

### ISSUE THREAD

**Issue by courtneygringras** [bcgov/SIMS#373]
Student application questions found here: https://hive.aved.gov.bc.ca/wiki/display/SIMS/Student+Application+Questions

Review Vivian's document
Organize into sections
Organization of groupings (dependencies)
Must be able to see connections of questions - dependencies

**Comment by courtneygringras** [comment]
[https://app.zenhub.com/files/319429847/4bbef9a3-d302-4bf1-835c-016b70d81cd9/download](https://app.zenhub.com/files/319429847/4bbef9a3-d302-4bf1-835c-016b70d81cd9/download)

**Comment by courtneygringras** [comment]
Merged the information from Danna and Vivian's documents.  Updated question list found here - https://hive.aved.gov.bc.ca/wiki/display/SIMS/Student+Application+Questions 

Grouped into sections - not necessarily the sections we will be using for the question order, but grouped them by function so we can ensure the the conditional elements are met and we understand all of the information required for the application.

Conditional fields are highlighted in BLUE. 

The major finding from this activity was the implications of combining FT & PT and how they determine BC Residency requirements. This will change the existing order used on the mockups already created. I have created ticket #433 to address determining the order of the questions based on conditional requirements.
